resources/cluster-essentials/templates/crd-init-oathkeeper.yaml

```diff
@@ -22,9 +22,28 @@ spec:
         - name: crd-rules
           mountPath: /etc/ory/crd
           readOnly: true
-        command: ["kubectl",  "apply", "-f", "/etc/ory/crd/rules.yaml"]
+        command:
+        - /bin/bash
+        - -c
+        - |-
+          set -e
```

Comment:
In general we should consider retries in all kubectl operastions in jobs. Cool example can be found here with timeout & until https://github.com/kyma-project/kyma/blob/master/installation/resources/tiller.yaml#L129
crafted by our bash hero @Demonsthere